backend/danswer/db/models.py

```diff
@@ -156,10 +156,15 @@ class IndexAttempt(Base):
         String(), default=None
     )  # only filled if status = "failed"
     time_created: Mapped[datetime.datetime] = mapped_column(
-        DateTime(timezone=True), server_default=func.now()
+        DateTime(timezone=True),
+        server_default=func.now(),
+        index=True,
```

Comment:
I'd think the using a B-tree for this is overall good despite the slower writes since its not really that slow

Reply:
I think you actually need to add a new migration to add this, adding this param alone won't do anything 😓